mxcube3/routes/qutils.py

```diff
@@ -537,11 +542,12 @@ def set_dc_params(model, entry, task_data):
     acq.path_template.base_prefix = params['prefix']
 
     full_path = os.path.join(mxcube.session.get_base_image_directory(),
-                             params.get('path', ''))
+                             params.get('subdir', ''))
```

Comment:
Ok thanks for the explanation. I was confused by all the paths and `.split()` and indexes (`[-1]`, `[1]`...)